coresimd/src/x86/i586/sse.rs

```diff
@@ -906,7 +906,7 @@ pub unsafe fn _mm_movemask_ps(a: f32x4) -> i32 {
            assert_instr(unpcklps))]
 // TODO: This function is actually not limited to floats, but that's what
 // what matches the C type most closely: (__m128, *const __m64) -> __m128
-pub unsafe fn _mm_loadh_pi(a: f32x4, p: *const f32) -> f32x4 {
+pub unsafe fn _mm_loadh_pi(a: f32x4, p: *const u8x8) -> f32x4 {
```

Comment:
I think this should be `p: *const f32x2` because this loads two floats from `p` into the upper two floats of `a`.

Reply:
Also, in case we can't agree on that, then I think `_m64` would have been better than `u8x8`.